tools/functional-tester/etcd-tester/tester.go

```diff
@@ -104,6 +104,11 @@ func (tt *tester) doRound(round int) (bool, error) {
 			plog.Printf("%s recovery error: %v", tt.logPrefix(), err)
 			return false, nil
 		}
+		plog.Printf("%s wait until cluster is fully alive", tt.logPrefix())
+		if err := tt.cluster.WaitHealth(); err != nil {
```

Comment:
at least, we can remove the 7 thing. making it to 3 or something seems more reasonable. right now the 7 seconds retry seems really strange.